Sources/OTel/Metrics/OTelMetricExporter/OTelNoOpMetricExporter.swift

```diff
@@ -14,7 +14,6 @@
 /// A metric exporter that ignores all operations, used when no metrics should be exported.
 ///
 /// - TODO: Why are we providing this?
```

Comment:
That's actually a good question. I think we can drop this one as well as the no-op span exporter since we're not aiming to be spec compliant anyways and don't need it as a default somewhere.